app/tests/test_search_endpoint.py

```diff
@@ -0,0 +1,155 @@
+import logging
+import sys
+
+from django.conf import settings
+from django.test import Client
+
+from tests.base_test import StacBaseTestCase
+from tests.data_factory import Factory
+from tests.utils import client_login
+
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler(sys.stdout)
+logger.addHandler(handler)
+
+
+API_BASE = settings.API_BASE
+
+
+class SearchEndpointTestCase(StacBaseTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        cls.factory = Factory()
+        cls.collection = cls.factory.create_collection_sample().model
+        cls.items = cls.factory.create_item_samples(
+            [
+                'item-1',
+                'item-2',
+                'item-switzerland',
+                'item-switzerland-west',
+                'item-switzerland-east',
+                'item-switzerland-north',
+                'item-switzerland-south',
+                'item-paris'
+            ],
+            cls.collection,
+            db_create=True,
+        )
+
+    def setUp(self):  # pylint: disable=invalid-name
+        self.client = Client()
+        client_login(self.client)
+        self.path = f'/{API_BASE}/search'
+
+    def test_query(self):
+        # get match
+        title = "My item 1"
+        query = '{"title":{"eq":"%s"}}' % title
+        response = self.client.get(
+            f"/{API_BASE}/search"
+            f"?query={query}&limit=100")
+        self.assertStatusCode(200, response)
+        json_data_get = response.json()
+        self.assertEqual(json_data_get['features'][0]['properties']['title'], title)
+
+        # post match
+        payload = """
+        {"query":
+            {"title":
+                {"eq":"My item 1"}
+            }
+        }
+        """
+        response = self.client.post(self.path, data=payload, content_type="application/json")
+        self.assertStatusCode(200, response)
+        json_data_post = response.json()
+        self.assertEqual(json_data_post['features'][0]['properties']['title'], title)
+
+        # compare get and post
+        self.assertEqual(len(json_data_post['features']), len(json_data_get['features']))
```

Comment:
FOR NEXT PR: would check the payload equality and not the size. Both payload response should be equal instead of the timestamp.